site/src/template.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Title
+description: Description
+template: standard
+---
+
+# Page title (H1)
+
+The first thing is the general description.
+
+Then the list of some general classes can be listed, if there are not many of them, or css style specific details
+
+_Example_
+
+The following spacing sizes are supported:
+
+- `s`
+- `m`
+- `l`
+- `none`
+
+Otherwise consider using tables to organize information.
+
+_Example. Table 1_
+
+| cssproperty | value                                                   | screensize |
+| ----------- | ------------------------------------------------------- | ---------- |
+| `margin`    | `xxs`, `xs`, `s`, `m`, `l`, `xl`, `xxl`, `xxxl`, `none` | `tablet`   |
```

Comment:
```md
| `<name>` | `<value>`, `<value>`, `<value>` | `<size>
```

Reply:
Updated

Reply:
Doesn't look like this one got addressed